iBudget/templates/emails/update_password.html

```diff
@@ -0,0 +1,15 @@
+<h2>Your password has been changed
+    <i>
```

Comment:
Same as previous.

Reply:
I will do as u said in the future.Thank u